docs/pages/03-best-practices.md

```diff
@@ -0,0 +1,52 @@
+## Best practices
+
+Animations can greatly improve an interface's UX, but it's important to follow some guidelines to not overdo it and deteriorate the user experience on your web-things. Following the following rules should provide a good start.
+
+### Meaningful animations
+
+You should avoid animating an element just for the sake of it. Keep in mind that animations should make an intention clear. Animations like attention seekers (bounce, flash, pulse, etc) should be used to bring the user's attention to something special in your interface and not only as a way to bring "flashiness" to it.
+
+Entrances and exit animations should be used to orientate what is happening in the interface, clearly signaling that it's transitioning into a new state.
+
+It doesn't mean that you should avoid adding playfulness to the interface, just be sure that the animations are not getting in the way of your user and that the page's performance is not affected by an exaggerated use of animations.
+
+### Don't animate large elements
+
+Avoid it as it won't bring much value to the user and will probably only cause confusion. Besides that, there is a good chance that the animations will be junky, culminating in bad UX.
+
+### Never animate root elements
+
+Animate the `<html/>` or `<body/>` tags is a no go as you'll just go through some browser weirdness.
```

Comment:
This language is a bit vague; let’s see if we can smarten it up (linking to corroborating evidence can help).